lib/LaTeXML/Post/MathML.pm

```diff
@@ -535,7 +535,14 @@ sub pmml_maybe_resize {
   my $depth  = $node->getAttribute('depth')   || ($parent && $parent->getAttribute('depth'));
   my $xoff   = $node->getAttribute('xoffset') || ($parent && $parent->getAttribute('xoffset'));
   my $yoff   = $node->getAttribute('yoffset') || ($parent && $parent->getAttribute('yoffset'));
-  if ($width || $height || $depth || $xoff || $yoff) {
+  my $role   = $node->getAttribute('role')    || ($parent && $parent->getAttribute('role'));
+  my $unstretchy = (($node->getAttribute('stretchy') || ($parent && $parent->getAttribute('stretchy'))) || 'yes') eq 'no';
+
+  # First, special case hack for stretchy arrows, with specified width
+  # Stretchiness (currently) only has effect within munder/mover!!!!!
+  if ($width && (($role || '') eq 'ARROW') && !$unstretchy) {    # SPECIAL CASE HACK
+    $result = ['m:mover', {}, $result, ['m:mspace', { width => $width }]]; }
```

Comment:
I wonder if a well-named `class` attribute for this special case won't be useful in the future. Would make it a little clearer for people inspecting the markup, and easy to check when it gets refactored as MathML Core improves.

Maybe `<mover class="ltx_math_stretched">` ?

Reply:
The `$unstretchy` check may not be booleany enough. If we go with strecthy="true" by default, and when explicit, then everything else is false (including invalid values), then:

```perl
  my $stretchy_attr = $node->getAttribute('stretchy') || ($parent && $parent->getAttribute('stretchy')) || "true";
  my $unstretchy = $strechy_attr neq "true";
```

Reply:
Somehow I knew you'd suggest that, so why didn't I just head you off by implementing it?  But actually, something like `\ltx_horizontally_stretched` might be even better; I was a little concerned about the code doing something weird with vertically stretching cases. (Alas, no visible distinction in MathML)

Reply:
Actually, if a class is going to be used to detect the "special case", then we can drop the test of `stretchy` completely.